checklists/alz_checklist.en.json

```diff
@@ -2480,22 +2480,78 @@
-            "text": "Connect default resource configurations to a centralized Azure Monitor Log Analytics workspace.",
+            "text": "Creating dedicated workspaces for Monitor and Secuity to to segregate ownership of data between operational and security teams and optimize costs.",
             "waf": "Security",
             "service": "Monitor",
             "guid": "e5f8d79f-2e87-4768-924c-516775c6ea95",
             "id": "G03.08",
-            "severity": "Medium",
+            "severity": "High",
             "link": "https://learn.microsoft.com/azure/azure-monitor/logs/design-logs-deployment"
         },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Implement SIEM solution to consolidate security data in a central location where it can be correlated across various services.",
+            "waf": "Security",
+            "service": "Entra",
+            "guid": "a56888b2-7e83-4404-bd31-b886528502d1",
+            "id": "G03.09",
+            "severity": "High",
+            "graph": "resources| where type == 'microsoft.operationalinsights/workspaces'| extend wsid = properties.customerId| project workspaceResourceId = tolower(id), name, wsid| join (resources| where type == 'microsoft.operationsmanagement/solutions'| where name has 'SecurityInsights'| extend workspaceResourceId = tostring(tolower(properties.workspaceResourceId))| project workspaceResourceId | summarize ResourceCount = count() by workspaceResourceId) on workspaceResourceId| extend RCount = iff(isnull(ResourceCount), 0, ResourceCount)| project-away ResourceCount| extend compliant = (RCount <> 0)",
+            "link": "https://learn.microsoft.com/en-us/azure/sentinel/connect-data-sources?tabs=azure-portal"
+        },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Facilitates proactive threat hunting and investigation via Sentinel - built-in threat detection mechanisms to detect and respond to security threats in real time.",
+            "waf": "Security",
+            "service": "Entra",
+            "guid": "edca434d-965f-43b2-84b0-d5d8449b5aaf",
+            "id": "G03.10",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/sentinel/overview?tabs=azure-portal"
+        },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Incorporate security alerts into your incident response system",
+            "waf": "Security",
+            "service": "Entra",
+            "guid": "13aad129-a2e5-4099-b738-9918f4d6a917",
+            "id": "G03.11",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/sentinel/relate-alerts-to-incidents"
+        },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Automate the response to security alerts",
```

Comment:
This needs to be more specific.  What, specifically, is the configuration that needs to be done.  See [What to contribute](https://github.com/Azure/review-checklists/blob/main/CONTRIBUTING.md#what-to-contribute) to get more specifics.

Reply:
I get your point, I can change it to: "Automate the response to security alerts, use a playbook and automation rule to automate incident response and remediate security threats."
in parallel, I do see other items in the list that are not specific, like this one: "Enforce centralized and delegated responsibilities to manage resources deployed inside the landing zone, based on role and security requirements"

Reply:
I think that is still pretty vague for the Landing Zone side.  It might make sense to have another checklist to review Security Response, but I don't think it should go into here.

Is there a specific part of CAF Ready that we are looking to include here?  If we are capturing what is reflected there, then I am OK with it, but if it isn't part of CAF Ready and is instead more of another part of CAF, then I think the other checklist might be good.